src/Pages/Notes/NotesPage.jsx

```diff
@@ -0,0 +1,11 @@
+import { Navbar } from "../../components/Navbar/Navbar";
+import { Sidebar } from "../../components/Sidebar/Sidebar";
```

Comment:
Suggestion : you can create index.js file in component folder and import these from there